docs/product-spec.md

```diff
@@ -0,0 +1,104 @@
+many have a completely manual process for processing absentee ballot
+requests, stuffing and mailing ballots out, and finally receiving,
+sorting, verifying, extracting, and scanning the returned ballots. A
+manual process that works for a few hundred ballots quickly falls over
+for a few thousand ballots, and establishing a more automated process
+in the 6 months we have left is a huge challenge for each small county
+to undertake.
+
+We propose VxMail, a set of tools to help small counties quickly scale
+up their vote-by-mail capacity.
+
+## Key Hypothesis
+
+A number of small and some medium-sized counties have a sufficiently
+manual process for a relatively small number of vote-by-mail ballots
+today. COVID-19 will cause, for them, a shift to vote-by-mail, with 10
+times more ballots, which makes the manual process no longer
+practical&mdash;because of insufficient speed and accuracy. With
+simple COTS hardware and software, we can enable 3 election staffers
+to handle 50,000 vote-by-mail ballots assuming at least 4 weeks for
+sending out ballots prior to election day and 2 weeks for tabulation.
+
+## VxMail Components
+
+VxMail includes:
+* a Mail Ballot Management System (MBMS) to send mail ballots to voters.
+* a Signature Verification System (SVS) to check signatures on incoming ballots.
+* tools to efficiently open envelopes and extract ballots.
+* a Ballot Scanning System (BSS) to scan and tabulate ballots.
+
+Only the MBMS is online for connecting with print&mail services, while
+the other systems are offline for security purposes.
+
+### Mail Ballot Management System: printing & mailing ballots
+
+VxMail includes a _Mail Ballot Management System (MBMS)_ where an
+election official can:
+
+* upload an election definition
+* upload a mailing list
+* proof ballots
+* order ballots to be printed and mailed
+* track ballot delivery to voters.
+* track ballot return from voters.
+
+All ballot printing, stuffing, and mailing is automated. Voters
+receives a ballot and return envelope, including a signature box.
+
+### Signature Verification System
+
+When ballots are received, VxMail lets election officials verify
+signatures as follows:
+
+* a state export of voter IDs and signatures is loaded into the VxMail Signature Verification System (SVS)
+* Ballot envelopes are scanned via the SVS scanner in small batches. On screen, expected and actual signatures are displayed for confirmation by election officials. If any of the batch are a non-match, SVS pauses while that ballot envelope is set aside. Scanning then continues, one small batch at a time.
+* When all ballots have been scanned, an export of successful and unsuccessful voter IDs is generated for upload into the state voter management system to record that those voters have cast a ballot.
+
+### Envelope Opening & Extraction
+
+Depending on the size of the county, an off-the-shelf automated
+envelope opener can be used. Alternatively, a simple envelope cutter
```

Comment:
Clarifying: "off-the-shelf and proven automatic envelope opener and letter extractor can be used, reducing the personnel needed for opening ballots." For smaller counties, affordable automatic letter openers are available